angular/src/app/pipes/date-ago/date-ago.pipe.ts

```diff
@@ -0,0 +1,39 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'dateAgo'
+})
+export class DateAgoPipe implements PipeTransform {
+
+  private intervals: {[key: string]: number} = {
+    'year': 60*60*24*365,
+    'month': 60*60*24*30,
```

Comment:
It is nice to see such a useful pipe in our application :) I'm super glad

Reply:
Thank you :). I'm so proud for that .... Feel free to use it :dancers: